tests/scripts/thread-cert/Cert_5_5_02_LeaderReboot.py

```diff
@@ -89,7 +89,7 @@ def test(self):
         self.assertEqual(self.nodes[ROUTER].get_state(), 'leader')
 
         self.nodes[LEADER].start()
-        self.simulator.go(5)
+        self.simulator.go(10)
```

Comment:
Is it necessary to change the wait time?
I think `5s` might be enough for the `Leader` to attach in theory.

Reply:
supposed to wait more time and see if packets not caught, change back to 5 sec